ssl/ech/ech_internal.c

```diff
@@ -11,11 +11,78 @@
 #include <openssl/ech.h>
 #include "../ssl_local.h"
 #include "ech_local.h"
+#include <openssl/rand.h>
+#include "../statem/statem_local.h"
+#include <internal/ech_helpers.h>
+#include <openssl/kdf.h>
 
 #ifndef OPENSSL_NO_ECH
 
+/*
+ * Strings used in ECH crypto derivations (odd format for EBCDIC goodness)
+ */
+/* "ech accept confirmation" */
+static const char OSSL_ECH_ACCEPT_CONFIRM_STRING[] = "\x65\x63\x68\x20\x61\x63\x63\x65\x70\x74\x20\x63\x6f\x6e\x66\x69\x72\x6d\x61\x74\x69\x6f\x6e";
+/* "hrr ech accept confirmation" */
+static const char OSSL_ECH_HRR_CONFIRM_STRING[] = "\x68\x72\x72\x20\x65\x63\x68\x20\x61\x63\x63\x65\x70\x74\x20\x63\x6f\x6e\x66\x69\x72\x6d\x61\x74\x69\x6f\x6e";
+
 /* ECH internal API functions */
 
+# ifdef OSSL_ECH_SUPERVERBOSE
+/* ascii-hex print a buffer nicely for debug/interop purposes */
+void ech_pbuf(const char *msg, const unsigned char *buf, const size_t blen)
```

Comment:
All functions that are not static must have `ossl_` prefix

Reply:
done (I think:-)